docs/reference/ilm/error-handling.asciidoc

```diff
@@ -3,13 +3,13 @@
 == Troubleshooting {ilm} errors
 
 When {ilm-init} executes a lifecycle policy, it's possible for errors to occur
-while performing the necessary index operations for a step. 
-When this happens, {ilm-init} moves the index to an `ERROR` step. 
-If {ilm-init} cannot resolve the error automatically, execution is halted  
+while performing the necessary index operations for a step.
+When this happens, {ilm-init} moves the index to an `ERROR` step.
+If {ilm-init} cannot resolve the error automatically, execution is halted
```

Comment:
For future PRs, in order to aid with backporting and reviewing, please have the formatting separately (separate PRs)

Reply:
Ok, I will revert them for now.

Reply:
Let's remove them and keep the scope of this PR as small and targeted as possible